docker/openproblems-r-extras/requirements.txt

```diff
@@ -1,2 +1,3 @@
 git+https://github.com/KrishnaswamyLab/harmonic-alignment#subdirectory=python
 git+https://github.com/theislab/scib
+anndata>=0.7.6,<0.8
```

Comment:
This should be set in `setup.py` since it's a requirement for all of the images, not just the extras